src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -166,14 +226,48 @@ public static int getChannelsNumberFromLibMediaInfo(String mediaInfoValue) {
 		}
 	}
 
+	public static int getBitsperSampleFromLibMediaInfo(String mediaInfoValue) {
+		if (isEmpty(mediaInfoValue)) {
+			LOGGER.warn("Empty value passed in. Returning default number 16.");
+			return 16;
+		}
+
+		// examples of libmediainfo  (mediainfo --Full --Language=raw file):
+		// Bit depth : 16
+		// Bit depth : 24
+		// Bit depth : / 24 / 24
+
+		int result = -1;
+		Matcher intMatcher = intPattern.matcher(mediaInfoValue);
+		while (intMatcher.find()) {
```

Comment:
I was not, but it's fine as it was what i have done first by myself, because it the more safe, compatible way. If by last, you mean at left end of the string.

Though i don't really believe you on that, because i think that it will choose the largest value, or a valid one, not the last one (except if it a correct value and the largest) ?